server/mdm/apple/vpp/refresh.go

```diff
@@ -0,0 +1,44 @@
+package vpp
+
+import (
+	"context"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/mdm/apple/itunes"
+)
+
+// RefreshVersions updatest the LatestVersion fields for the VPP apps stored in Fleet.
+func RefreshVersions(ctx context.Context, ds fleet.Datastore) error {
+	apps, err := ds.GetAllVPPApps(ctx)
```

Comment:
Could this be a huge result set, should we do this in batches? Or worry about that optimization later?

Reply:
Ah yeah I raised this in slack;

I don't see customers having more than a few hundred VPP apps max, so I think this is OK for now, but am open to batching if needed! 

@noahtalerman do you know if we should expect customers to have very large numbers of VPP apps in Fleet?

Reply:
Sounds like it could be an over optimization for now. Let's keep it like this and if reports roll in that it's slow or too memory intensive we can deal with it then. 👍

Reply:
Great call!